.github/workflows/images.yaml

```diff
@@ -0,0 +1,129 @@
+name: Build
+
+on:
+  push:
+    tags:
+      - '*'
+      - '!JENKINS*'
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Get the Ref
+      id: get-ref
+      uses: ankitvgupta/ref-to-tag-action@master
+      with:
+        ref: ${{ github.ref }}
+        head_ref: ${{ github.head_ref }}
+
+    - name: Build wmagent image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/wmagent/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/wmagent
+
+    - name: Build workqueue image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/workqueue/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/workqueue
+
+    - name: Build reqmon image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/reqmon/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/reqmon
+
+    - name: Build t0_reqmon image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/t0_reqmon/Dockerfile.dist
```

Comment:
We still do not have this `t0_reqmon` package in PyPi. I need to review it, but I think the Dockerfile will have to:
```
RUN pip install reqmon-$TAG.tar.gz
```

we need to investigate if:
* t0_reqmon is even expected to be packaged in WMCore
* if so, if the package is the same as `reqmon`. It could be that it's during service start up that we decide which CouchApps to use and whatnot.